### ISSUE TITLE
Reduce build times and frontend JS size by removing excess dependencies.

### ISSUE THREAD

**Issue by gak** [TBD54566975/ftl#2147]
Currently it takes ~10s to build release on a M2, and 16.1 MB to serve in dev.

All FTL builds and most CI rely on this install.

**Comment by gak** [comment]
Looks like this might be adding a lot of deps that we don't need: https://github.com/jsonnext/codemirror-json-schema/issues/125

**Comment by gak** [comment]
- https://github.com/shikijs/shiki/issues/658

**Comment by gak** [comment]
Removing codemirror deps and temporarily showing a blank div brings the build down to 3s and 1.7 MB.

**Comment by wesbillman** [comment]
@gak I just tried this locally on this branch and on `main` and didn't notice a massive difference:
```
this-branch: just build-all  22.41s user 9.17s system 144% cpu 21.893 total
main: just build-all  28.29s user 9.80s system 139% cpu 27.295 total
```

Definitely reduces the size though. I'm ok with pulling this out for now, but we might have to clean up the new version of the form a bit since this might be the most used feature on the console 😂

**Comment by wesbillman** [comment]
Also, without json-schema validation in the UI it might be really difficult for users to know what fields they need to send on a request. Maybe we need to solve that in another way, but there currently isn't a great way to see what should and should not be there. 🤔

**Comment by gak** [comment]
I am bringing it back, just with less deps. It will at least save the size of the deploy at least.

**Comment by wesbillman** [comment]
Nice! Sounds great 👍

**Comment by wesbillman** [comment]
I think this was "fixed" with #2183

**Comment by gak** [comment]
Ah of course, closing!
